examples/syscall-steal.c

```diff
@@ -43,6 +43,19 @@
 
 #if defined(CONFIG_KPROBES)
 #define HAVE_KPROBES 1
+#if defined(CONFIG_X86_64)
+/* For x86 architecture, the syscall table cannot be used to invoke a syscall
+ * after commit 1e3ad78 since v6.9. This commit has been backported to long 
+ * term stable kernels, like v5.15, v6.1, v6.6 and v6.8. In this case, use a 
+ * hook on the syscall entry instead to intercept the syscall. 
+ * For more details, see https://stackoverflow.com/questions/78599971/hooking-syscall-by-modifying-sys-call-table-does-not-work/78607015#78607015.
```

Comment:
The rationale for using Kprobe on the x86 architecture is informative. Could you move the description out of this file? Specifically, the text should appear in the LaTeX script so that readers can gain a deeper understanding before examining the kernel module source code.